scipy/stats/contingency.py

```diff
@@ -271,3 +272,122 @@ def chi2_contingency(observed, correction=True, lambda_=None):
                                    lambda_=lambda_)
 
     return chi2, p, dof, expected
+
+def association(arr, chi2_stat, stat="cramer"):
+    """Calculates degree of association between variables
+    that are nominal or greater.
+
+    Allows for specification of one of four related methods,
+    Tschuprow's T, Pearson's Contingency Coefficient, Cramer's V and Phi.
+
+    Parameters
+    ----------
+    arr : array-like
+        The array of observed values
+    chi2_stat : float
+        The chi-squared statistic.
+    stat : {"cramer", "tschuprow", "pearson", "phi"} (default = "cramer")
+        The association test statistic.
+
+    Returns
+    -------
+    value : float
+        Value of the test statistic
+
+    Notes
```

Comment:
Could you "reflow" the text in the Notes section?  Some of the lines break much earlier than necessary.